src/pages/BalanceNew/StuckOraib/index.tsx

```diff
@@ -0,0 +1,53 @@
+import { ReactComponent as StuckOraibridge } from 'assets/icons/stuck_oraib.svg';
+import TokenBalance from 'components/TokenBalance';
+import styles from './style.module.scss';
+import loadingGif from 'assets/gif/loading.gif';
+import { RemainingOraibTokenItem } from './useGetOraiBridgeBalances';
+import { TooltipIcon } from './TooltipBridgeToken';
+
+interface Props {
+    handleMove: () => Promise<void>;
+    loading: Boolean;
+    remainingOraib: RemainingOraibTokenItem[]
+}
+
+export default function StuckOraib({ handleMove, loading, remainingOraib }: Props) {
+    if (!remainingOraib.length) {
+        return <></>;
+    }
+    return (
+        <div className={styles.bridgeBalances}>
+            <StuckOraibridge className={styles.stuckIcon} />
+            {loading ?
+                <img src={loadingGif} alt='loading-gif' width={30} height={30} />
+                :
+                <div className={styles.stuckText} onClick={handleMove}>Move Stuck Token</div>
+            }
+            <TooltipIcon
+                placement="bottom-end"
+                content={remainingOraib.map(token => {
+                    const { Icon } = token
+                    const oraibDecimals = 18;
```

Comment:
This also needs to be edited. A rule of thumb is that you should not hard code any value. If you are planning to hard code anything, its best to ask others first. We should get the oraib decimals from filtering the token info from the list in bridgeTokens.ts file